### ISSUE TITLE
Tapping on a message in the topic narrow will load the message topic into compose box

### ISSUE THREAD

**Issue by kunall17** [zulip/zulip-mobile#1273]
I find the old native app very usable because of this thing as currently if someone sends a new message in a topic view it will go into (no topic) which is quite confusing if they haven't used the toggle compose tools, on tapping on message it would also toggle the topic view so people might understand it more.

I know we'll be doing a re work of compose box soon, but let's start from here! 

Let's upvote/downvote react if you agree/disagree?

**Comment by timabbott** [comment]
Yeah, if you're in a topic narrow, we should definitely definitely set the topic to the current topic.
